### ISSUE TITLE
Problem trying to rewire function within same file

### ISSUE THREAD

**Issue by mexin** [asapach/babel-plugin-rewire-exports#15]
Im having trouble using rewire$functionName when I try to use it on a function invocation within the same file. 

Example:
```
// fileA.js
const functionA = function() {
    const b = functionB('Test');
    return 'This is a ' + b;
};
    
const functionB = function(text) {
    return text.toUpperCase();
}
    
export { functionA, functionB }

```
on my mocha test I have the following

```
import { functionA, rewire$functionB, restore } from './fileA'

describe('Should return text', function() {
    let funcB;
    before(function () {
        rewire$functionB(funcB = sinon.stub());
    });
    after(function() {
        restore();
    });
    it('Should return test', function(){
        funcB.returns('not a test');
        const val = functionA();
        expect(val).to.be.equals('This is not a test');
    });
});
```

Every time I run the test, it never stubs functionB.

Is there something I missed?

Thanks a lot!

**Comment by sergei-startsev** [comment]
@mexin constants cannot be changed, you can set [unsafeConst](https://github.com/asapach/babel-plugin-rewire-exports/blob/master/README.md#options) option or just replace `cost` by `let` in your code.

**Comment by mexin** [comment]
> @mexin constants cannot be changed, you can set [unsafeConst](https://github.com/asapach/babel-plugin-rewire-exports/blob/master/README.md#options) option or just replace `cost` by `let` in your code.

I forgot to mention that I'm using `unsafeConst`, only on this particular case seems its not working properly.

Thanks

**Comment by asapach** [comment]
@mexin, just to confirm, could you please change `const functionB` to `let functionB` and see if that works? I that helps I'll see if I can make `unsafeConst` work better.

**Comment by asapach** [comment]
As a workaround you can rewrite it like this and it should work:
```js
export const functionA = function() {
  const b = functionB('Test');
  return 'This is a ' + b;
};

export const functionB = function(text) {
  return text.toUpperCase();
};
```

**Comment by asapach** [comment]
@mexin, I've released a new version - please try `v1.2.0`
